src/assets/scripts/client/commandParser/argumentValidators.js

```diff
@@ -228,3 +229,20 @@ export const holdValidator = (args = []) => {
         }
     }
 };
+
+/**
+ * Checks that `args` has one value that corresponds to a valid squawk
+ *
+ * @function squawkValidator
+ * @param args {array}
+ * @return {string|undefined}
+ */
+export const squawkValidator = (args = []) => {
+    if (args.length !== 1) {
+        return ERROR_MESSAGE.SINGLE_ARG_LENGTH;
+    }
+
+    if (REGEX.TRANSPONDER_CODE.exec(args[0]) == null) {
```

Comment:
Two things here:
- why not use `.test()` instead of `exec()`?
- please use strict equals `===` for comparisons